js/interface.js

```diff
@@ -1402,7 +1430,16 @@ $('#show-access-rules').click(function () {
             }
 
             return 'Specific users<br />' + _.map(Object.keys(rule.allow.user), function (key) {
-              return '<code>' + key + ' = ' + rule.allow.user[key] + '</code>';
+              var operation = rule.allow.user[key];
+
+              if (typeof operation === 'string') {
```

Comment:
I've lost track of the overall data structure because we're a few PRs away from the original. Can you explain what this `if` does?

Reply:
this was to support an old syntax, I've removed it.